### ISSUE TITLE
Connect you fails to add new contact from WhatsApp

### ISSUE THREAD

**Issue by zoroya1** [Bnyro/ConnectYou#136]
### Steps to reproduce

1. Press **Add** button when a message from unknown number arrives
2. It shows 2 options
3. **Adding to existing contact** works fine (it correctly copies the number in number field)
4. But **create new contact** fails to copy and paste that number in number field in connect you app.

### Expected behavior

While creating new contact, number should be automatically get pasted in number field

### Actual behavior

Instead, number field remains black while creating new contact and contact fails to get added in address book.

### Connect You version

3.0

### Android version

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
That probably is a duplicate of #129 and #130?

**Comment by zoroya1** [comment]
Yes, number 129 is exactly what I was trying to tell, my bad i only saw open issues, forgot to see closed issues which can mention same issue. Sorry :)
Thanks for this app and awesome work 🥰
